### PR TITLE
Suntoh/feat/login page

### PR DESCRIPTION
**Pull request by Suntoh**
## What I did

- registration form for new user
- send new user data to the database

## Note

required finalizing which user data will be used in the database
ex. gender, bio ,country

### CODE REVIEW

File: apps/server/src/controllers/user.ts [hunk 1]
```diff
@@ -61,6 +61,10 @@ export const loginUser = async (req: Request, res: Response) => {
       return res.status(200).json({ error: 'Username not found' });
     }
 
+    if (!user.password) {
+      return res.status(500).json({ error: 'User password is not set' });
+    }
```
Comment (oatkup1a) [inline apps/server/src/controllers/user.ts:66]: Can you provide more explanation why this is needed in loginUser?

**Comment by oatkup1a** [comment]
LGTM🥳 (for backend part)
Note: For finalizing user data in database, you can consult with me later in discord. For now, I'll follow in types.ts for your request.

**Review by oatkup1a — APPROVED** [review]
LGTM🥳 (for backend part)
Note: For finalizing user data in database, you can consult with me later in discord. For now, I'll follow in types.ts for your request.

File: apps/server/src/controllers/user.ts (hunk 1 above)
Comment (Suntoh) [reply apps/server/src/controllers/user.ts:66]: i think it's for when password is not sent. It's use to handle that case

**Pull request merged**